addons/missileguidance/functions/fnc_onFired.sqf

```diff
@@ -109,19 +109,30 @@ private _args = [_this,
             [
                 getNumber ( _config >> "seekerAngle" ),
                 getNumber ( _config >> "seekerAccuracy" ),
-                getNumber ( _config >> "seekerMaxRange" )
+                getNumber ( _config >> "seekerMaxRange" ),
+                getNumber ( _config >> "seekerMinRange" )
             ],
             [ diag_tickTime, [], [], _lastKnownPosState]
         ];
 
-
-// Run the "onFired" function passing the full guidance args array
-private _onFiredFunc = getText (_config >> "onFired");
+private _onFiredFunc = getText (configFile >> QGVAR(SeekerTypes) >> _seekerType >> "onFired");
+TRACE_1("",_onFiredFunc);
+if (_onFiredFunc != "") then {
+    _args call (missionNamespace getVariable _onFiredFunc);
+};
+        
+_onFiredFunc = getText (configFile >> QGVAR(AttackProfiles) >> _attackProfile >> "onFired");
 TRACE_1("",_onFiredFunc);
 if (_onFiredFunc != "") then {
     _args call (missionNamespace getVariable _onFiredFunc);
 };
         
+// Run the "onFired" function passing the full guidance args array
+_onFiredFunc = getText (_config >> "onFired");
+TRACE_1("",_onFiredFunc);
+if (_onFiredFunc != "") then {
+    _args call (missionNamespace getVariable _onFiredFunc);
+};
         
 // Reverse:
```

Comment:
Remove comment if unused.

Reply:
What do you mean?

Reply:
Lines 126 to 158 are comments. If the code is not used it is better to remove it.